cvat/apps/iam/rules/tests/generators/users_test.gen.rego.py

```diff
@@ -46,13 +46,9 @@ def read_rules(name):
 
 
 def RESOURCES(scope):
-    if scope == "list":
-        return [None]
-    else:
-        return [
-            {"id": random.randrange(300, 400), "membership": {"role": role}} for role in ORG_ROLES
-        ]
-
+    return [
+        {"id": random.randrange(300, 400), "membership": {"role": role}} for role in ORG_ROLES
+    ]
```

Comment:
You don't need these changes because resource should always be `None` when scope == "list"